docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -1465,6 +1465,24 @@ How to **investigate**:
 - Check if ingesters are processing too many records, and they need to be scaled up (vertically or horizontally).
 - Check actual error in logs to see whether the `-ingest-storage.kafka.wait-strong-read-consistency-timeout` or the request timeout has been hit first.
 
+### MimirKafkaClientBufferedProduceBytesTooHigh
+
+This alert fires when the Kafka client buffer, used to write incoming write requests to Kafka, is getting full.
+
+How it **works**:
```

Comment:
Skipping this suggestion, because it's consistent with the rest of the file. You can open a PR to change it everywhere if it's something worth to you.